apps/sms/locales/sms.en-US.properties

```diff
@@ -315,6 +315,10 @@ contact-photo.ariaLabel = Contact photo
 unread-thread.ariaLabel = Unread
 has-draft.ariaLabel = Has draft
 is-draft.ariaLabel = Draft
+optionsButton.ariaLabel = Options
+settingsButton.ariaLabel = Settings
+composeLink.ariaLabel = Compose
+callButton.ariaLabel = Call
```

Comment:
OK I understand what you say;

Because the buttons are actions, how about having always a verb:
* show options
* show settings
* compose (message) <= optional; but I have an issue here, is that the button is really to compose a message to a recipient we choose after; as opposed to choosing an existing conversation and composing a message to _this_ recipient. I'd like a verb to show this difference. Maybe "New message" ? "new" is not used as a verb these days according to wiktionary :) I'd accept either "compose", "compose message" or "new message", unless you find something better, so please pick your prefered one !
* call <= it's pretty clear we want to call this recipient